packages/cli/src/commands/build.ts

```diff
@@ -1,8 +1,7 @@
 import {Command} from '@oclif/command'
 import {build} from '@blitzjs/server'
 
-// eslint-disable-next-line import/no-default-export
```

Comment:
just checking, because of the eslint-disable comments, do we intend to export these as `default`?

Reply:
nope, that's just legacy. moving forward we'd like everything to be named exports